opencti-platform/opencti-front/src/private/components/data/import/ImportFilesContent.tsx

```diff
@@ -0,0 +1,350 @@
+import { graphql } from 'react-relay';
+import React, { useState } from 'react';
+import ImportMenu from '@components/data/ImportMenu';
+import { ImportFilesContentQuery, ImportFilesContentQuery$variables } from '@components/data/import/__generated__/ImportFilesContentQuery.graphql';
+import Tooltip from '@mui/material/Tooltip';
+import IconButton from '@mui/material/IconButton';
+import { DeleteOutlined, GetAppOutlined } from '@mui/icons-material';
+import DialogContent from '@mui/material/DialogContent';
+import DialogContentText from '@mui/material/DialogContentText';
+import DialogActions from '@mui/material/DialogActions';
+import Button from '@mui/material/Button';
+import Dialog from '@mui/material/Dialog';
+import { ImportFilesContentLines_data$data } from '@components/data/import/__generated__/ImportFilesContentLines_data.graphql';
+import { ImportFilesContentFileLine_file$data } from '@components/data/import/__generated__/ImportFilesContentFileLine_file.graphql';
+import { useFormatter } from '../../../../components/i18n';
+import { APP_BASE_PATH } from '../../../../relay/environment';
+import useQueryLoading from '../../../../utils/hooks/useQueryLoading';
+import { emptyFilterGroup } from '../../../../utils/filters/filtersUtils';
+import { usePaginationLocalStorage } from '../../../../utils/hooks/useLocalStorage';
+import DataTable from '../../../../components/dataGrid/DataTable';
+import Transition from '../../../../components/Transition';
+import useApiMutation from '../../../../utils/hooks/useApiMutation';
+import Breadcrumbs from '../../../../components/Breadcrumbs';
+import { UsePreloadedPaginationFragment } from '../../../../utils/hooks/usePreloadedPaginationFragment';
+import { deleteNode } from '../../../../utils/store';
+
+export const WorkbenchFileLineDeleteMutation = graphql`
+  mutation ImportFilesContentFileLineDeleteMutation($fileName: String) {
+    deleteImport(fileName: $fileName)
+  }
+`;
+
+export const workbenchLineFragment = graphql`
+  fragment ImportFilesContentFileLine_file on File {
+    id
+    entity_type
+    name
+    uploadStatus
+    lastModified
+    lastModifiedSinceMin
+    objectMarking {
+      id
+      definition_type
+      definition
+      x_opencti_order
+      x_opencti_color
+    }
+    metaData {
+      mimetype
+      list_filters
+      labels
+      messages {
+        timestamp
+        message
+      }
+      errors {
+        timestamp
+        message
+      }
+      creator {
+        name
+      }
+      entity_id
+      entity {
+        ... on AttackPattern {
+          name
+        }
```

Comment:
would this be sufficient ? if only for display

```
... on StixObject {
          representative {
            main
          }
        }
```

Reply:
Done